frontend/src/pages/onboarding/index.tsx

```diff
@@ -112,7 +116,10 @@ export default function Onboarding() {
     resolver: yupResolver(formSchema),
     mode: 'onSubmit',
   });
-  const { handleSubmit, trigger } = methods;
+  const { handleSubmit, trigger, watch } = methods;
+
+  // for debugging
+  useWatchForm(watch);
```

Comment:
Now I think bout it this function can add a suffix DevDebug or Soemthing like that, so it's obvious

Reply:
agree!